tests/test_core.py

```diff
@@ -1803,8 +1803,8 @@ async def test_services_call_return_response_requires_blocking(
             return_response=True,
         )
     assert (
-        str(exc.value)
-        == "A non blocking service call with argument blocking=False can't be used together with argument return_response=True"
+        str(exc.value) == "A non blocking service call with argument blocking=False "
```

Comment:
Can we wrap the string inside a parenthesis and make a newline before writing the string so both lines of the string have the same indentation?